domain/src/main/java/com/architectcoders/module/domain/remote_models/Characters/Characters.kt

```diff
@@ -0,0 +1,11 @@
+package com.architectcoders.module.domain.remote_models.Characters
+
+data class Characters(
+    val code: Int?,
+    val status: String?,
+    val copyright: String?,
+    val attributionText: String?,
+    val attributionHTML: String?,
+    val etag: String?,
+    val `data`: Data?
```

Comment:
@joseantoniocampillo  could you rename `data` to characterData?

Reply:
done